dashboard/lib/policies/lti.rb

```diff
@@ -193,9 +193,9 @@ def self.issuer_accepts_resource_link?(issuer)
     ['Canvas'].include?(issuer_name(issuer))
   end
 
-  # Force Schoology through iframe mitigation flow
+  # Force Schoology and Canvas through iframe mitigation flow
   def self.force_iframe_launch?(issuer)
-    ['Schoology'].include?(issuer_name(issuer))
+    %w[Schoology Canvas].include?(issuer_name(issuer))
```

Comment:
I would recommend moving these values to a constant:
```ruby
class Policies::Lti
  ISSUERS = [
    CANVAS = 'Canvas'.freeze,
    SCHOOLOGY = 'Schoology'.freeze
    LTI_V1 = 'LMS'.freeze
  ].freeze

  IFRAME_MIGRATON_FLOW_ISSUERS = [CANVAS, SCHOOLOGY].freeze
end
```

Reply:
@nicklathe @carl-codeorg Is it okay to use an i18n value in `Policies::Lti.issuer_name`, considering that different languages may have different values?
https://github.com/code-dot-org/code-dot-org/blob/72ebc1ed1c85750768c5e9ac272ff11681ee404f/dashboard/lib/policies/lti.rb#L162
https://github.com/code-dot-org/code-dot-org/blob/72ebc1ed1c85750768c5e9ac272ff11681ee404f/dashboard/config/locales/en.yml#L328
[fr-FR.yml](https://media.githubusercontent.com/media/code-dot-org/code-dot-org/72ebc1ed1c85750768c5e9ac272ff11681ee404f/dashboard/config/locales/fr-FR.yml): `lti_v1: Plateforme d'apprentissage`

Reply:
> I would recommend moving these values to a constant:
We could move these to a constant, however this is just a temporary solution. Since we only support Canvas and Schoology, and we are sending both through the iframe jailbreak flow, we'll likely remove this check in the future anyways. I think leaving it as is for now is totally fine.

> Is it okay to use an i18n value in Policies::Lti.issuer_name, considering that different languages may have different values?

Issuer won't ever be translated, as this is a claim from the JWT returned from either Schoology or Canvas, stating who the issuing party of the JWT is. It is part of the OIDC spec, and not user facing translatable text.